docs/Charting-Concepts/VerticalBarChart.md

```diff
@@ -0,0 +1,445 @@
+1. The legend is rendered using the Legends component, which is passed an array of ILegend objects. The chart can also display callout labels for each bar, which are rendered using SVG text elements.
+1. The chart can be customized with various props, including barWidth, colors, theme, lineLegendText, yAxisTickCount, and more. The chart also supports accessibility features, including aria labels and keyboard navigation.
+- **Error scenarios**
+  The Vertical bar chart handles the following error scenario:
+  - Empty data: If the data passed to the chart component is empty, the chart will not render and a message will be narrated to the user. \_isChartEmpty functions handles that scenario.
+- **Localization aspects**
+  The component has a culture prop, which is used to set the culture for the chart. This prop is used to format the x-axis labels and the callout values based on the specified culture. Currently, vertical bar chart provides localization only for the x-axis ticks if they are numbers.
+- **Testing**
+  The manual and component tests for Vertical bar chart has been completed and unit testing are in progress. Following is the current code coverage for Vertical bar chart:
+
+    ![VerticalBarChart3.png](../assets/images/VBC3.png)
+
+1. Component Tests:
+   1. Work item: <https://uifabric.visualstudio.com/iss/_workitems/edit/7435>
+   1. Test plan: <https://github.com/microsoft/fluentui/blob/master/packages/react-charting/docs/TestPlans/VerticalBarChart/ComponentTests.md> 
+1. Unit Tests:
+   1. Work item: <https://uifabric.visualstudio.com/iss/_workitems/edit/7436>
+1. Manual Tests:
+   1. Work item: <https://uifabric.visualstudio.com/iss/_workitems/edit/8604>
+1. Accessibility Tests:
+   1. Work item: <https://uifabric.visualstudio.com/iss/_workitems/edit/7434>
+- **Accessibility**
+  Vertical bar chart is tested for fast pass accessibility.
+  Link to the FAST pass tool: <https://accessibilityinsights.io/docs/web/getstarted/fastpass/> 
+  Our charts have elaborate accessibility support. The charts are WCAG 2.1 MAS C compliant for accessibility.
+  Consumers can define their own aria labels for each point by setting the callOutAccessibilityData property.
+- **Theming**
+  The palette for vertical bar chart is set from the "theme" prop as passed to the component during rendering. Both light and dark themes are supported and users can create there own theme too. [Ref6](https://github.com/microsoft/fluentui/wiki/Theming)  and [Ref7](https://github.com/microsoft/fluentui/wiki/How-to-apply-theme-to-Fluent-UI-React-components)  explains theming in detail.
+- **Debugging**
+  The detailed steps on debugging has been given in [Debugging](https://uifabric.visualstudio.com/iss/_wiki/wikis/iss.wiki?wikiVersion=GBwikiMaster&pagePath=/Fluent%20React%20Charting/Debugging).
```

Comment:
Update the links pointing to wiki to contrib site

Reply:
updated the debugging link which is available in the contrib repo